### PR TITLE
Fix api

### LINKED ISSUE #28: Check public API
Hi Johannes @jo-mueller ,

before releasing the first version to pypi, could you please carefully check that the public API is as you want it to be? At the moment, all functions in `biapol_utilities.label` are also available in `biapol_utilities` directly because of [this line](https://github.com/BiAPoL/biapol-utilities/blob/main/biapol_utilities/__init__.py#L19). Hence, these two lines do the same:

```
biapol_utilitis.label.compare_labels(a, b)
biapol_utilitis.compare_labels(a, b)
```

Is this intentional? If so, why? If not, consider either renaming "label" to "_label" or remove the line linked above, depending on how you want the public API to be.

Thanks!

Best,
Robert

### PR DESCRIPTION
**Pull request by jo-mueller**
## Description

I changed function imports in __init__.py on subpackage level from `from subpackage import *` to `from . import subpackage`. This forces the user to call function from a certain subpackage as `biau.subpackage.function(...)`, the call `biau.function(...)` wiill no longer work.

## Type of change

* [x] Bug-fix
* [ ] New feature
* [x] Breaking change
* [ ] Documentation update

## References 

Closes #28 

## Tests

* [x] I adapted existing tests, because
* [ ] I added new tests to cover the code change
* [x] All tests pass with my change

## Final checks

* [x] My change is the minimal possible work for the desired feature/fix
* [ ] I updated the documentation where necessary to cover the change
* [ ] I rebuilt the documentation page to document the change, if necessary

### CODE REVIEW

**Comment by jo-mueller** [comment]
@haesleinhuepf I fixed the imports that relied on old API calls. The tests do not cover calls in notebooks so that's something to keep in mind.

File: biapol_utilities/__init__.py [hunk 1]
```diff
@@ -16,7 +16,8 @@
 __version__ = "0.0.1"
 
 # from .measure import *
-from .label import *
+from . import label
```
Comment (haesleinhuepf) [inline biapol_utilities/__init__.py:19]: I don't think this is necessary as the subfolder is called "label" anyway. Would you mind trying it without that line?

**Review by haesleinhuepf — APPROVED** [review]
Looks good to me @jo-mueller . Furthermore, if you write `closes #28` in your PR description, the issue 28 will be automatically closed when merging this PR. Try it out ;-)

Furthermore, if the code still works without that one line I mentioned. Feel free to merge. :-)

File: biapol_utilities/__init__.py (hunk 1 above)
Comment (jo-mueller) [reply biapol_utilities/__init__.py:19]: > I don't think this is necessary as the subfolder is called "label" anyway. Would you mind trying it without that line?

For me it seems like it is necessary. Commenting the line throws an error that the subpackage `label` is not found when I run the respective test.

**Comment by jo-mueller** [comment]
> Looks good to me @jo-mueller . Furthermore, if you write closes #28 in your PR description, the issue 28 will be automatically closed when merging this PR. Try it out ;-)

Fancy! I changed it further up 👍

**Comment by zoccoler** [comment]
Hi Johannes @jo-mueller,

I just tested this branch and the subpackage way of calling works (ran files in tests folder). Direct calling indeed no longer works then.

Just the last test (`test_match_labels.py`) yields this error (divide by zero?):

```
...biapol-utilities\biapol_utilities\label\_intersection_over_union.
py:50: RuntimeWarning: invalid value encountered in true_divide
  iou = overlap / (n_pixels_pred + n_pixels_true - overlap)
```

Also a minor thing: some places I see `import biapol_utilities as biau` and others it is `biao`

**Comment by jo-mueller** [comment]
Hi Marcelo @zoccoler ,

thanks for pointing this out. I changed the inputs accordingly. The errormessages are just warnings that are usually thrown in `intersection_over_union_matrix`. It would be nice to to suppress this somehow - but that's for a different PR.

Feel free to merge this :)

**Comment by jo-mueller** [comment]
@zoccoler @haesleinhuepf , I fixed the conflicts with the main branch so I think this could be merged as well.

**Review by zoccoler — APPROVED** [review]

**Pull request merged**